packages/calling/src/Events/impl/index.ts

```diff
@@ -12,6 +12,8 @@ export class Eventing<T extends EventMap> extends (EventEmitter as {
   new <T extends EventMap>(): TypedEmitter<T>;
 })<T> {
   /**
+   * @event
+   *
    * @param event - TODO.
```

Comment:
These TODOs are showing up in the docs. Can we fix  this as well?

Reply:
Removed